### PR TITLE
Rework training page

### PR DESCRIPTION
**Pull request by bobturneruk**

### CODE REVIEW

**Comment by bobturneruk** [comment]
I'm particularly interested in feedback from @annakrystalli and @twinkarma, as leaders in this area.

**Comment by twinkarma** [comment]
I think it's looking good. I'll link to the DL courses from the Deep Leaning page as well. And we now run 2 introductory DL courses now. One with Nvidia DLI and one is our own. I'll go in to add courses and modify the descriptions once this is merged in.

**Comment by willfurnass** [comment]
This looks really good!  We can add in tagging/filtering in time but this is a great starting point.  

Other things we might want to add at some point:
 - HPC Carpentry (after we've run something for the N8)
 - https://github.com/RSE-Sheffield/hi-perf-ipynb (hasn't been updated in a couple of years; needs reviewing before we say it's something we can offer)
 
@mondus Think any CUDA-related resources should be added at this point prior to Pete starting to run NVIDIA CUDA courses?

**Review by willfurnass — APPROVED** [review]
Made some suggestions in a stand-alone comment but I don't think any are blockers to getting this merged.

File: pages/training/courses/ACCE.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: Reproducible Research Data and Project Management in R
+layout: page
+type: text
+---
+
+**Prerequisite skills:** Degree in a quantitative discipline, or equivalent.
```
Comment (bobturneruk) [inline pages/training/courses/ACCE.md:7]: @annakrystalli - is this OK?

File: pages/training/courses/git_Hero.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+title: git & GitHub through GitKraken - from Zero to Hero!
+layout: page
+type: text
+---
+
+**Prerequisite skills:** Some programming experience.
```
Comment (bobturneruk) [inline pages/training/courses/git_Hero.md:7]: Is this right @annakrystalli?

**Comment by bobturneruk** [comment]
Thanks @willfurnass and @twinkarma - I've put a couple of questions to @annakrystalli - I'll hold off merging a bit.

File: pages/training/courses/ACCE.md (hunk 1 above)
Comment (annakrystalli) [reply pages/training/courses/ACCE.md:7]: Yup looks good!

File: pages/training/courses/git_Hero.md (hunk 2 above)
Comment (annakrystalli) [reply pages/training/courses/git_Hero.md:7]: Yup LGTM

**Review by annakrystalli — APPROVED** [review]

**Pull request merged**